frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield.jsx

```diff
@@ -60,6 +60,7 @@ export default class ExpressionEditorTextfield extends React.Component {
     onError: PropTypes.func.isRequired,
     startRule: PropTypes.string.isRequired,
     onBlankChange: PropTypes.func,
+    helpTextTarget: PropTypes.instanceOf(Element).isRequired,
```

Comment:
`ExpressionEditorTextfield.jsx` is only used by 2 components:
1. `ExpressionWidget.jsx`
2. `ExpressionPopover.jsx`